pkg/utils/setup_test.go

```diff
@@ -23,21 +25,32 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client/fake"
 
 	vpcresourcesv1beta1 "github.com/aws/amazon-vpc-resource-controller-k8s/apis/vpcresources/v1beta1"
+	"github.com/aws/amazon-vpc-resource-controller-k8s/pkg/aws/ec2/api"
 )
 
 var (
-	testSA                *corev1.ServiceAccount
-	testPod               *corev1.Pod
-	testScheme            *runtime.Scheme
-	testClient            client.Client
-	testSecurityGroupsOne []string
-	testSecurityGroupsTwo []string
-	helper                SecurityGroupForPods
-	name                  string
-	namespace             string
-	saName                string
+	testSA                        *corev1.ServiceAccount
+	testPod                       *corev1.Pod
+	testScheme                    *runtime.Scheme
+	testClient                    client.Client
+	testSecurityGroupsOne         []string
+	testSecurityGroupsTwo         []string
+	testSecurityGroupNamesToIdMap map[string]string
+	helper                        SecurityGroupForPods
+	name                          string
+	namespace                     string
+	saName                        string
 )
 
+// TODO: Use the mocks and actually correctly mock this
```

Comment:
Should this be updated? As I see there is a mock EC2 API helper.

Reply:
Removed